appdata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+ <id>com.github.falkTX.Carla</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-2.0</project_license>
+ <name>Carla</name>
+ <summary>Audio Plugin Host</summary>
+ <description>
+  <p>Carla is a fully-featured audio plugin host, with support for many audio drivers and plugin formats.</p>
+ </description>
+ <content_rating type="oars-1.1" />
+ <launchable type="desktop-id">com.github.falkTX.Carla.desktop</launchable>
+ <update_contact>supgesu_AT_gmail.com</update_contact>
+ <url type="homepage">https://kx.studio/Applications:Carla</url>
+ <url type="bugtracker">https://github.com/falkTX/Carla/issues</url>
+ <releases>
+   <release version="2.4.1" date="2021-10-15" />
+ </releases>
```

Comment:
missing `oars`, `screenshots`. so it won't validate.

Reply:
also the file should be submitted upstream.

Reply:
added screenshots and submitted it upstream: https://github.com/falkTX/Carla/pull/1535
not sure about oars, since there is a xml node for it. Any additional info needed?

Reply:
nevermind about oars, I missed seeing it here. It's good. Sorry abou that.